x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/attribute_selector.test.tsx

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { shallow, ShallowWrapper } from 'enzyme';
+
+import { EuiComboBox, EuiFieldText } from '@elastic/eui';
+
+import { AttributeSelector } from './attribute_selector';
+import { ANY_AUTH_PROVIDER, ANY_AUTH_PROVIDER_OPTION_LABEL } from './constants';
+
+const handleAttributeSelectorChange = jest.fn();
+const handleAttributeValueChange = jest.fn();
+const handleAuthProviderChange = jest.fn();
+
+const baseProps = {
+  attributeName: 'An Attribute',
+  attributeValue: 'Something',
+  attributes: ['a', 'b', 'c'],
+  availableAuthProviders: ['ees_saml', 'kbn_saml'],
+  selectedAuthProviders: ['ees_saml'],
+  elasticsearchRoles: ['whatever'],
+  multipleAuthProvidersConfig: true,
+  disabled: false,
+  handleAttributeSelectorChange,
+  handleAttributeValueChange,
+  handleAuthProviderChange,
+};
+
+describe('AttributeSelector', () => {
+  it('renders', () => {
+    const wrapper = shallow(<AttributeSelector {...baseProps} />);
+
+    expect(wrapper.find('[data-test-subj="attributeSelector"]')).toBeDefined();
```

Comment:
I think this test will pass regardless of whether the element exists or not... I think `find` always returns something.

```suggestion
    expect(wrapper.find('[data-test-subj="attributeSelector"]').exists()).toBe(true);
```